.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug report
+description: You've found a bug on the documentation website
+labels: ['bug']
+body:
+  - type: dropdown
+    attributes:
+      label: What browser are you using?
+      description: We do not support Internet Explorer 11.
+      options:
+        - 'Android browser'
+        - 'Baidu Browser'
+        - 'Chrome for Android'
+        - 'Chrome'
+        - 'Edge'
+        - 'Firefox for Android'
+        - 'Firefox'
+        - 'Internet Explorer'
+        - 'KaiOS'
+        - 'Opera Mini'
+        - 'Opera Mobile'
+        - 'Opera'
+        - 'QQ Browser'
+        - 'Safari on iOS'
+        - 'Safari on macOS'
+        - 'Samsung Internet'
+        - 'UC browser for Android'
+        - 'Other'
```

Comment:
I don't know if we really want such a large list of "browser options" in the dropdown?

We could go for something lighter like this:

- Chrome
- Firefox
- Edge
- Safari iOS
- Safari macOS
- Internet Explorer
- Other

We would need another single-line text input field to specify the "other browser", I think?